woocommerce/woocommerce-ios
# Reapply updates from "marked as read" notice PR (#579)

This PR is a dupe of #579 — it re-adds changes from that.

## Testing

1. Queue up a notification on your test site (new order or new review)
2. Build and run the app
3. Open the notifications tap
4. Wait for the new notification(s) to load
5. Tap the mark all as read button in the navbar **(1st time)**
    - [ ] Verify you see the "marked all notifications as read" notice appear
6. Queue up another notification on your test site (new order or new review)
7. On the notification tab pull to refresh and wait for the new notification to appear
8. Tap the mark all as read button in the navbar **(2nd time)**
    - [ ] Verify you see the "marked all notifications as read" notice appear
9. Queue up another notification on your test site (new order or new review)
10. On the notification tab pull to refresh and wait for the new notification to appear
11. Tap the mark all as read button in the navbar **(3rd time)**
    - [ ] Verify you do **not** see notice appear

Feel free to repeat steps 9-11...you should not see the notice again.
